OOP/Assignment22/SurveyQuestionList.java

```diff
@@ -0,0 +1,51 @@
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+//this class will hold all question which are being read from a file
+public class SurveyQuestionList
+{
+	public SurveyQuestionList()
+	{
+		questions = new ArrayList<String>();
+		questionTypeList = new ArrayList<String>();
+		questionOption = new ArrayList<String>();
+		setupQuestions();
+	}
+
+	List<String> questions;
```

Comment:
Better way to declare variables on top, then write code like constructor to use them.